starcluster/node.py

```diff
@@ -254,7 +266,12 @@ def tags(self):
         return self.instance.tags
 
     def add_tag(self, key, value=None):
-        return self.instance.add_tag(key, value)
+        try:
```

Comment:
Why not have the retry logic in `add_tag`?   I assume the failure in alias occurs here.   Funny that the line
self.tags, which also accesses instance,  doesn't barf.  

Web says something about region not being set when getting this error, but it seems like we always set this to us-east-1

Reply:
Do we get a stack trace on this error?